app/frontend/src/features/userPage/UserPlace.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { SmokingLocation, User } from "../../pb/api_pb";
+import { makeStyles, Typography } from "@material-ui/core";
+import TextBody from "../../components/TextBody";
+import { smokingLocationLabels } from "../../constants";
+
+const useStyles = makeStyles((theme) => ({
+  list: {
+    listStyle: "none",
+    padding: 0,
+    marginBlockStart: 0,
+    //workaround for https://github.com/cssinjs/jss/issues/1414
+    //still present despite upgrading to 10.5.0
+    marginBlockEnd: `${theme.spacing(1)}px`,
+  },
+  listItem: {},
+  hostingPreferenceResponse: {
+    fontWeight: "bold",
+  },
+  subheading: {
+    fontSize: "1.25rem",
+  },
+}));
+
+export default function UserPlace({ user }: { user: User.AsObject }) {
+  const classes = useStyles();
+
+  const preferences = [
+    [
+      !!user.lastMinute,
+      "Last minute requests ok?",
+      user.lastMinute?.value ? "Yes" : "No",
+    ],
+    [!!user.maxGuests, "Max guests", user.maxGuests?.value.toString()],
+    [
+      !!user.multipleGroups,
+      "Multiple groups ok?",
+      user.multipleGroups?.value ? "Yes" : "No",
+    ],
+    [
+      !!user.wheelchairAccessible,
+      "Wheelchair accessible",
+      user.wheelchairAccessible?.value ? "Yes" : "No",
+    ],
+    [
+      user.smokingAllowed !== SmokingLocation.SMOKING_LOCATION_UNKNOWN &&
+        user.smokingAllowed !== SmokingLocation.SMOKING_LOCATION_UNSPECIFIED,
+      "Smoking allowed?",
+      smokingLocationLabels[user.smokingAllowed],
+    ],
+    [
+      !!user.acceptsKids,
+      "Kids allowed?",
+      user.acceptsKids?.value ? "Yes" : "No",
+    ],
+    [
+      !!user.acceptsPets,
+      "Pets allowed?",
+      user.acceptsPets?.value ? "Yes" : "No",
+    ],
+  ];
+
+  return (
+    <>
+      <Typography variant="h3" className={classes.subheading}>
+        Hosting preferences
+      </Typography>
+      <ul className={classes.list}>
+        {preferences.map((pref, index) =>
```

Comment:
```suggestion
        {preferences.map(([hasSetPreference, label, value], index) =>
```
I think destructuring `pref` into their own variables like that might make this slightly more readable.

Reply:
But creating those variables within the iteration (without using your `preferences` solution) would be even better, I think.

Reply:
@amdwit how do you mean?

Reply:
Well, I think it is now the best.